

I8042_KBD_IRQ and I8042_AUX_IRQ are not constant on sparc64.

Signed-off-by: Andrew Morton <akpm@osdl.org>
---

 25-sparc64-akpm/drivers/input/serio/i8042.c |    2 --
 1 files changed, 2 deletions(-)

diff -puN drivers/input/serio/i8042.c~input-serio-dynamic-allocation-fix-2 drivers/input/serio/i8042.c
--- 25-sparc64/drivers/input/serio/i8042.c~input-serio-dynamic-allocation-fix-2	2004-06-19 23:19:49.691165872 -0700
+++ 25-sparc64-akpm/drivers/input/serio/i8042.c	2004-06-19 23:19:49.695165264 -0700
@@ -79,7 +79,6 @@ struct i8042_values {
 };
 
 static struct i8042_values i8042_kbd_values = {
-	.irq 		= I8042_KBD_IRQ,
 	.disable	= I8042_CTR_KBDDIS,
 	.irqen 		= I8042_CTR_KBDINT,
 	.mux		= -1,
@@ -87,7 +86,6 @@ static struct i8042_values i8042_kbd_val
 };
 
 static struct i8042_values i8042_aux_values = {
-	.irq 		= I8042_AUX_IRQ,
 	.disable	= I8042_CTR_AUXDIS,
 	.irqen		= I8042_CTR_AUXINT,
 	.mux		= -1,
_
